backend/src/main/resources/db/migration/V3__Task.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE task(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+	name varchar(256),
+	description TEXT,
+    is_required BOOLEAN DEFAULT true,
```

Comment:
aren't all tasks required?

Reply:
Again, this is if the task is required to complete its milestone

Reply:
when would a task not be required to complete a milestone?

Reply:
it was specifically discussed in the requirements before. I believe their reasoning was something like this eaxmple, we want the student to get feedback on their resume, and there may be multiple was to do that, see different advisors, attend resume building event, etc. but they don't need to do all of them to "finish" the milestone